loadbalancer.tf

```diff
@@ -66,10 +66,10 @@ resource "aws_lb_target_group" "keycloak-target-group" {
     interval            = "300"
     protocol            = "HTTP"
     matcher             = "200"
-    timeout             = "3"
+    timeout             = "10"
```

Comment:
10 seconds seems high, so I looked at the TargetResponseTime metrics for the production load balancer over the last 48 hours: 

![Max AWS_ApplicationELB TargetResponseTime Maximum](https://github.com/user-attachments/assets/c6714803-87c2-441b-a485-8bdaf1f78b8d)

We do occasionally see a 10 second delay between when the load balancer makes a request to a Keycloak ECS request and when it gets a response back. Which is certainly a long time, and we should investigate that, but for the moment it justifies why this value needs to be this high. Note that this wasn't during a deploy, just normal operation of the production Keycloak cluster, so I'd be curious to see metrics surrounding a deploy as well.